server.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express")
+const app = express();
+
+
+app.listen(3000,()=>{
+    console.log("server running in port 3000")
+})
+
+app.get('/ping',(req,res)=>{
```

Comment:
Consider adding a catch-all route handler at the end of your routes to handle 404 errors. This can be done with `app.use('*', (req, res) => res.status(404).send('Not Found'));`. [medium]